tests/test_dockers.py

```diff
@@ -152,11 +152,14 @@ def test_paddlenlp(self):
             "paddlenlp", "fill-mask", "PaddleCI/tiny-random-bert"
         )
         self.framework_docker_test(
-            "paddlenlp", "conversational", "PaddlePaddle/plato-mini"
+            "paddlenlp", "conversational", "PaddleCI/tiny-random-plato-mini"
+        )
+        self.framework_docker_test(
+            "paddlenlp", "zero-shot-classification", "PaddlePaddle/tiny-random-bert"
```

Comment:
Is that correct ? Or
```suggestion
            "paddlenlp", "zero-shot-classification", "PaddleCI/tiny-random-bert"
```
Maybe ?